solr/CHANGES.txt

```diff
@@ -29,7 +29,7 @@ Optimizations
 
 Bug Fixes
 ---------------------
-(No changes)
+* SOLR-11191: Splitting shards now routes child-docs with their _root_ field when available so they maintain parent relationship. (Zack Kendall)
```

Comment:
> FWIW I view this as an Improvement but it's debatable.

IMO the coin could tip based on existing documentation; with no prior acknowledgement of the limitations of these features, I would expect child-docs and shard-splitting to work together without breaking, so I consider not doing so to be a bug.

> Also by putting this here, it will not be back-ported to 9.x. But wouldn't we want it in the next 9.x release?

Can you elaborate or point me to instructions to differentiate the options here. I would like to get it in 9.x. I was guessing that after getting it into mainline latest, I would then PR it separately to 9 branch and at that point move the change note to under 9, but I could use some guidance on expected workflow.

Reply:
Simply move this entry to the corresponding part of the 9.8 heading.  I will take care of back-porting to branch_9x after merging this; that's not for you to worry about.  Usually back-ports don't get PRs.

Reply:
Cool, done.